src/Controls/src/Core/Platform/Android/BottomNavigationViewUtils.cs

```diff
@@ -109,8 +144,13 @@ static async Task SetMenuItemIcon(IMenuItem menuItem, ImageSource source, IMauiC
 				source,
 				context.Context);
 
-			if (menuItem.IsAlive() && result is not null)
-				menuItem.SetIcon(result.Value);
+			if (menuItem.IsAlive())
+			{
+				if (result is not null)
+					menuItem.SetIcon(result.Value);
+				else
+					menuItem.SetIcon(null);
```

Comment:
this can be made much simpler:
```suggestion
				menuItem.SetIcon(result?.Value);
```
A single line will have the same effect.